### ISSUE TITLE
Yogar-CBMC fails with Segmentation fault

### ISSUE THREAD

**Issue by PavelAndrianov** [yinliangze/yogar-cbmc#1]
Hi!
I am trying to run yogar-cbmc (sv-comp version) on [cil.i.zip](https://github.com/yinliangze/yogar-cbmc/files/793674/cil.i.zip), and it fails:
> We use CBMC version 4.9 64-bit linux
file yogar-tmp/cil.s.i: Parsing
Converting
Type-checking cil.s
Generating GOTO Program
Adding CPROVER library
file <command-line> line 0: <command-line>:0:0: warning: "__STDC_VERSION__" redefined
file <command-line> line 0: <built-in>: note: this is the location of the previous definition
Segmentation fault

The command line is
> ./yogar-cbmc cil.i

The input file is compiled by gcc, thus, seems, it is correct (but rather complicated).

**Comment by yinliangze** [comment]
Thanks for providing this example. It may be a bug of yogar-cbmc. Can you provide the original .c source file? The cil.i file is too complicated. We will try to solve this problem in next month.

**Comment by PavelAndrianov** [comment]
We have no single .c file. The example is constructed with our [framework](http://linuxtesting.org/) from a Linux driver. You may found corresponding .c files, for example, in [lxr](http://lxr.free-electrons.com/). Use the hints in the line directives: they contains a suffix, like "drivers/ata/libata-pmp.c". Moreover, we add our generated code (environment model, model functions). I can provide the distinct "model" (generated) files, but the code is not simpler then in the final cil.i. 
To sum up, if you have any concrete questions about the code, I am glad to help, but I think the dozens of source files just confuse you (note, they are also complicated). Anyway, I am ready to provide all of auxiliary files, which we obtain at every stage of instrumentation, if you really need them :)
